docs/private-dns/api/reference.md

```diff
@@ -10,14 +10,99 @@ toc_max_heading_level: 4
     https://api.adguard-dns.io/static/swagger/openapi.json to markdown using
     https://swagger-markdown-ui.netlify.app/.
 
+    Changelog is from here:
+    https://api.adguard-dns.io/static/api/CHANGELOG.md
+
     If you want to change it, ask the developers to change the OpenAPI spec.
 -->
 
 ## AdGuard DNS API
```

Comment:
Should we remove these lines?

```
## AdGuard DNS API

DNS API documentation
```

Reply:
It's auto-generated, any change here will make it much more complex to apply updates